tests/base.py

```diff
@@ -1,4 +1,5 @@
 import distutils
+import distutils.spawn
```

Comment:
Do we need this import?

Reply:
tl:dr; no, I'll remove that change.

Looks like on python3.9 that I have laying around in an older machine:
```
Python 3.9.2 (default, Feb 28 2021, 17:03:44)
[GCC 10.2.1 20210110] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import distutils; distutils; distutils.__version__
<module 'distutils' from '/usr/lib/python3.9/distutils/__init__.py'>
'3.9.2'
>>> distutils.spawn
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AttributeError: module 'distutils' has no attribute 'spawn'
>>> import distutils.spawn
>>> distutils.spawn; distutils.spawn.find_executable
<module 'distutils.spawn' from '/usr/lib/python3.9/distutils/spawn.py'>
<function find_executable at 0x7fc388fd1ee0>
```

It appears:

- in python3.9, there is a distutils package coming from Python, and in that version, `distutils.__init__py` does not import spawn, hence the need for that import
- if you have installed setuptools, then the above works:

```
Python 3.9.16 (main, Dec  7 2022, 10:16:11)
[Clang 14.0.0 (clang-1400.0.29.202)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> import distutils; distutils
>>> distutils
<module 'distutils' (/Users/fperrin/src/pyangbind/env/lib/python3.9/site-packages/setuptools/_distutils/__init__.py)>
>>> distutils.spawn.find_executable
<function find_executable at 0x10188bd30>
```

- distutils will disappear in python3.12: https://github.com/python/cpython/pull/99061
- the syntax `install_requires = file: requirements.txt` in setup.cfg requires a somewhat recent version of setuptools (https://github.com/pypa/setuptools/pull/3253), so between this and the previous point, users have to install/upgrade setuptools _anyway_, so I can drop the import you mention

Reply:
Awesome, thanks for digging that.